contracts/src/cosmos/precompile/examples/Fundraiser.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: MIT
+//
+// Copyright (c) 2023 Berachain Foundation
+//
+// Permission is hereby granted, free of charge, to any person
+// obtaining a copy of this software and associated documentation
+// files (the "Software"), to deal in the Software without
+// restriction, including without limitation the rights to use,
+// copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the
+// Software is furnished to do so, subject to the following
+// conditions:
+//
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+// OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+// HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+// WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+// OTHER DEALINGS IN THE SOFTWARE.
+
+pragma solidity ^0.8.17;
+
+import {IBankModule} from "../bank.sol";
+
+/**
+ * @dev LiquidStaking is a contract that allows users to delegate their Base Denom to a validator
+ * and receive a liquid staking token in return. The liquid staking token can be redeemed for Base
+ * Denom at any time.
+ * Note: This is an example of how to delegate Base Denom to a validator.
+ * Doing it this way is unsafe since the user can delegate more straight through precomile.
+ * And withdraw via the precompile.
+ */
+contract Fundraiser {
+    address public owner;
```

Comment:
can remove this field and use the `Owner` contract

Reply:
what is "the Owner contract"?